roles/openshift_master/tasks/main.yml

```diff
@@ -172,14 +172,9 @@
   register: start_result
   notify: Verify API Server
 
-- name: Check for non-HA master service presence
-  command: systemctl show {{ openshift.common.service_type }}-master.service
-  register: master_svc_show
-  changed_when: false
-
-- name: Stop and disable non-HA master when running HA
+- name: Stop and disable non HA master when running HA
   service: name={{ openshift.common.service_type }}-master enabled=no state=stopped
-  when: openshift_master_ha | bool and 'LoadState=not-found' not in master_svc_show.stdout
```

Comment:
It was intentional but possibly mistaken, I think at the time I had reasoned that the service would be guaranteed to be there. Can dig more tomorrow but let me know if anyone knows otherwise. Harmless to restore it in any case.